code/client/CarPhysics.h

```diff
@@ -0,0 +1,21 @@
+#define TOML_EXCEPTIONS 0 // NOTE(beau): I don't want exceptions lmao
+#include "toml++/toml.h"
+
+#define FILEPATH "configs/CarPhysics.toml"
+#define ASSETS_FILEPATH "../../code/assets/configs/CarPhysics.toml"
```

Comment:
the serializer doesn't seem to write back to the main version controlled CarPhysics.toml file. I change a value, press t to read it (works fine) then I press s to write it to the main file (the one not in the 'out' folder) and I don't notice any change in the file (even though the console did say it was serializing the right values.
I fixed it by changing the ASSETS_FILEPATH to "../../../../assets/configs/CarPhysics.toml", but I wanted to run it by you before

Reply:
OH RIGHT aight so your file structure looks different than mine, I believe there's like "out" and "debug" folders in the path to the executable which is why you have to go back two more folders. Here's what my build folder looks like:

![image](https://user-images.githubusercontent.com/63375392/212158594-dd3d2a2b-1763-4361-8b0b-27b63677bcdf.png)

I'll add some conditional compilation to make it work for the windows filepath - hopefully you and elise have the same build dir structure.

Reply:
Sounds good, here is an idea of what my file path looks like to get to the runtime file path
C:\Users\oswei\source\repos\SamuelGarm\CPSC-585\code\out\build\x64-Debug\client\configs
and the regular file path
C:\Users\oswei\source\repos\SamuelGarm\CPSC-585\code\assets\configs